RobotCrawler/src/test/java/parser/ParserUtils.java

```diff
@@ -0,0 +1,25 @@
+package parser;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ParserUtils {
+
+    public static Document readHtmlFile(String baseUrl, String fileRelativePath) throws Exception {
+        String content = new String(Files.readAllBytes(createFileFromRelativePath(fileRelativePath)));
+        Document document = Jsoup.parse(content);
+        document.setBaseUri(baseUrl);
```

Comment:
Maybe throw more specific exception? Or write in javadoc what type of exceptions can be throw by jsoup.